corral/vcs/svn.pony

```diff
@@ -1,6 +1,8 @@
 use "files"
 
 primitive SvnVcs is Vcs
+  """Placeholder for Subversion VCS"""
```

Comment:
I think this should be 3 lines but the style guide isn't clear. @Theodus we should clarify that.